pages/a/max-internship-at-crate.html

```diff
@@ -0,0 +1,36 @@
+title: Internships: Max at Crate
+author: Max Liko
+description: During the hackathon we have been busy creating our own Kubernetes template.
+created: 2016-08-19
+tags:
+
+*To end his internship at Crate, we asked Max to contribute to our blog and tell us about the experience of working at Crate. At 18 years old he attends a technical school in Austria and while internships are sometimes mandatory, the company is free to choose. Luckily - he chose us.*
+
+
+"It would be fun they said, just an other internship they said", but they were so wrong (at least in respect to "just an internship") but let's start at the beginning.
+
+![Max cooking for the Dornbirn office.](/static/images/0816/max-cooking.jpg)
+
+
+After my successful application as an intern I started my six weeks at Crate's Dornbirn office on the 11th of July.
+I was not sure what to expect but the welcoming email raised my expectations and piqued my interest.
```

Comment:
This is a "teaser". What was in this welcoming email that made is so fascinating? We should create a more obvious link to the "awesome to the Max" image at the end.